### DISCUSSION TITLE
How to execute command without pushing to the jumplist?

### DISCUSSION THREAD

**Discussion by jpcrs** [vscode-neovim/vscode-neovim#1024] [Q&A]
I would like to know if it's possible to remap some commands to execute it without altering the vscode jumplist. Like `<C-d>`, `<C-u>`, and `<C-6>`.

**Comment by ttytm** [comment]
Hey @jpcrs. Would it be possible to be more specific maybe even give a specific example for what you want to achieve? This would give a less abstract, good foundation to work with

**Comment by jpcrs** [comment]
Hey @tobealive, sorry for not being more specific with my question, I wasn't sure about how to formulate it.

I usually change my vim/neovim config to make some commands not push to the jumplist, using the keepjumps cmd. For example:

```
nnoremap } :<C-u>execute "keepjumps norm! " . v:count1 . "}"<CR>
nnoremap { :<C-u>execute "keepjumps norm! " . v:count1 . "{"<CR>
```

This way I can use `{` and `}` multiple times without making my jumplist bloated. This is useful in some scenarios, for example when I Jump to a method definition, scroll through the file, and then go back to my previous location with `<C-o>`.

Apparently `:keepjumps` doesn't work on vscode-neovim, I guess because it's using the vscode back functionality instead of the neovim jumplist.

I would like to know if it's possible to remap some commands to execute it without altering the jumplist.
